### PR TITLE
Math-Magician: Events

### PR DESCRIPTION
**Pull request by Kaiserabbas**
- Created a new directory called logic inside your project
- Copied the files calculate.js and operate.js into logic/ directory
- Installed `big.js `using NPM

- Implemented click event handlers using the math logic from calculate.js and operate.js in React components.
- All calculator buttons work according to their functionality.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,77 +1,68 @@
...
+    );
+  };
+
   return (
     <div className="calculator">
-      <div className="display">0</div>
+      <div className="display">
+        {' '}
+        {calculatorData.total || calculatorData.next || '0'}
+      </div>
```
Comment (V-Blaze) [inline src/components/Calculator.js]: - You have done great so far :+1: :+1: Implementing events, but there are a few changes you need to make before moving to the next milestone. :smile: :small_airplane:  

- [ ] Looking at the video below, you can see that when I enter a number, it shows on the screen, but after selecting an operator, the numbers that follow are not shown on the screen. this is not proper as the user is meant to see what he/she is entering on the screen. I'll suggest you update the display logic. :+1: :smile: :wave: 

- - Video reference:

https://github.com/Kaiserabbas/math-magician/assets/107632780/71864a6f-4ef0-47ee-b53f-7d42e898688a

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Kaiserabbas 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
- Handles state with `useState`👋 
-  Readme looks very professional ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Kaiserabbas) [reply src/components/Calculator.js]: I have updated the logic that after selecting an operator, the numbers that follow are displayed properly.

**Review by Gambit142 — APPROVED** [review]
Hi @Kaiserabbas 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Great use of the logic directory ✔️ 
- Handles states with `useState` ✔️ 
- Implemented click events for all buttons ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**